lib/grid_game.js

```diff
@@ -62,6 +63,12 @@ var GridGame = {
     this.keyboard = new GridGame.classes.keyboard();
     this.keyboard.init();
   },
+
+  init_obstacles: function() {
+    for(var i=0; i < this.rock_count; i++){
```

Comment:
That's really interesting! I wouldn't have expected that to make any difference. May as well change that round, then.